runtime/compiler/aarch64/codegen/ARM64PrivateLinkage.cpp

```diff
@@ -422,7 +422,7 @@ void TR::ARM64PrivateLinkage::createPrologue(TR::Instruction *cursor)
    //
    cursor = loadStackParametersToLinkageRegisters(cursor);
 
-   setJittedMethodEntryPoint(cursor);
+   TR::Instruction *entryPointInstruction2 = cursor;
```

Comment:
Why not call this `beforeJittedMethodEntryPointInstruction` to make it a bit clearer?